integreat_cms/cms/utils/external_calendar_utils.py

```diff
@@ -28,6 +28,16 @@
 from integreat_cms.cms.utils.content_utils import clean_content
 
 
+class ImportResult:
+    # pylint: disable=too-few-public-methods
+    """
+    Datatype for the result of `import_event`
+    """
+
+    def __init__(self, number_of_errors: int):
+        self.number_of_errors = number_of_errors
+
+
```

Comment:
```suggestion
@dataclasses.dataclass(frozen=True, kw_only=True)
class ImportResult:
    """
    Datatype for the result of `import_event`
    """
    number_of_errors: int
```

This is very much a nitpick so feel free to ignore, but this is an ideal candidate for [dataclasses](https://docs.python.org/3/library/dataclasses.html). It would also get rid of the pylint warning.